lib/routes/controller.js

```diff
@@ -12,30 +12,46 @@ class Controller {
         this.extendedPerms = extendedPerms;
     }
 
-    get(path, handler, ...perms) {
-        if (this.extendedPerms && perms.length > 0) {
-            this.app.get(path, requirePerms(perms), handler.bind(this));
+    get(path, handler, permission) {
+        if (this.extendedPerms && permission) {
+            this.app.get(
+                path,
+                requirePermission(permission),
+                handler.bind(this)
+            );
```

Comment:
Instead of repeating this in all methods maybe we could extract the permissio-check in a helper function. 

```js
_checkPermission(permission) {
    if (this.extendedPerms && permission) {
        return requirePermission(permission)
	} else {
        return (req, res, next) => next();
    }
}
```

then we could simply call it:

```js
this.app.post(path, this.checkPermission(permission), handler.bind(this));
```